src/core/Dropdown/Dropdown/Dropdown.test.tsx

```diff
@@ -209,6 +208,5 @@ describe('Dropdown', () => {
 });
 
 test('CSS export', () => {
-  const css = cssFromBaseStyles(baseStyles);
-  expect(css).toEqual(expect.stringContaining('color'));
+  expect(baseStyles.join('')).toEqual(expect.stringContaining('color'));
 });
```

Comment:
I vote to remove this whole test as it's value seems to be very low and vague.

Reply:
Removed from both Dropdown and Language menu.